docs/topic-guides/working-with-starcounter/installation.md

```diff
@@ -1,42 +1,16 @@
 # Installation
 
-Starcounter can be installed in two different modes namely `Developer Mode` and `Production Mode`
-
- Following section describes these modes:
-
-## Developer Mode
-
-This is the default installation mode where the  `Starcounter Server Service`  can be started on-demand, typically when you start a Starcounter Application. 
+Installing Starcounter is straight forward and just like installing any other software application. Following sections guide you through the installation process:
```

Comment:
I think the sentence "just like installing any other software application" is somewhat ambiguous. I would prefer something more precise like: "You install Starcounter by downloading and running an installer. The following sections describes how to install Starcounter and the different options you have during the installation". 

Does that make sense? In my mind, installing a software application means running `sudo apt-get install [application name]`, and this is different than that.